src/autoBatchNode.js

```diff
@@ -0,0 +1,86 @@
+
+const Endpoint = require('./endpoint')
+const _ = require('lodash')
```

Comment:
(You don't use it here so it can be removed, but just FYI in 2021 using Lodash is pretty much always an antipattern)